qiskit/circuit/tools/pi_check.py

```diff
@@ -104,9 +104,9 @@ def normalize(single_inpt):
             if power[0].shape[0]:
                 if output == "qasm":
                     if ndigits is None:
-                        str_out = "{}".format(single_inpt)
+                        str_out = f"{single_inpt}"
                     else:
-                        str_out = "{:.{}g}".format(single_inpt, ndigits)
+                        str_out = f"{single_inpt:.{ndigits}g}"
```

Comment:
I didn't realize you could nest variables in an fstring like this for a formatter. That's a neat trick.

Reply:
yeah, not my trick, but i thought it was cool, too! :)